packages/search-input/src/SearchResultsMenu/LoadingOption/LoadingOption.styles.ts

```diff
@@ -0,0 +1,24 @@
+import { css, keyframes } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const loadingOptionStyles = css`
+  display: flex;
+  align-items: center;
+  gap: ${spacing[2]}px;
+  padding-block: ${spacing[1]}px;
+`;
+
+const loadingSpinner = keyframes`
+  from {
+    transform: rotate(0deg);
+  } 
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+export const loadingOptionIcon = css`
+  height: 16px;
+  width: 16px;
+  animation: ${loadingSpinner} 1.5s linear infinite;
+`;
```

Comment:
A `TODO` comment or ticket to make sure we update this to the new loading animation later might be nice

Reply:
I feel like we'll do an audit with the loading animation project coming up